geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientUpdateMessageImpl.java

```diff
@@ -1559,7 +1559,8 @@ public boolean isFull() {
   /**
    * Basically just a HashMap<String, Integer> but limits itself to the CqNameToOp interface.
```

Comment:
Should this comment be updated to match the change in map type?

Reply:
Can do.